dashboard/config/linux/upstream-apparmor-kasan.config

```diff
@@ -273,19 +275,19 @@ CONFIG_AIO=y
 CONFIG_IO_URING=y
 CONFIG_ADVISE_SYSCALLS=y
 CONFIG_MEMBARRIER=y
+CONFIG_KCMP=y
+CONFIG_RSEQ=y
+# CONFIG_DEBUG_RSEQ is not set
+CONFIG_CACHESTAT_SYSCALL=y
```

Comment:
If it's a new syscall, good to note in #533, or describe it if simple enough.

Reply:
Moved to #533 for now